tests/fuzz/s2n_certificate_extensions_parse_test.c

```diff
@@ -128,10 +128,6 @@ int LLVMFuzzerTestOneInput(const uint8_t *buf, size_t len)
         GUARD(s2n_pkey_zero_init(&public_key_out));
         s2n_pkey_type pkey_type;
 
-        /* Add cert to chain twice to reach codepaths that need 2+ certs */
```

Comment:
This comment suggests we should be validating a cert chain with two certs in it. That makes sense, bc we only expect extensions on the first cert in the chain.

* Are we using a cert chain with two certs?
* Why did calling this twice break things?

Reply:
* Why did calling this twice not break things previously?
* Are we guaranteed that no s2n connection will call this twice?

Reply:
Questions addressed offline.

Reply:
Questions resolved offline.